assets/base.css

```diff
@@ -3182,8 +3182,8 @@ details-disclosure > details {
 
 .animate--zoom-in > img,
 .animate--zoom-in > svg {
-  transition: transform var(--duration-short) linear;
-  transform: scale(var(--zoom-in-ratio));
+  transition: scale var(--duration-short) linear;
+  scale: var(--zoom-in-ratio);
```

Comment:
Sorry.. I'm seeing a shadow issue that I don't see with ambient motion selected or on the image banner PR. It affects iwt but I expect it's from something added on that PR. Can we see if there's anything we can do to avoid that?

https://screenshot.click/16-26-o8uak-0l6yh.mp4

Reply:
Should be good to re reviewed 👍 I was reusing a `clip-path` from the fixed image styling in the image banner and that was unnecessary

Reply:
That does fix the image with text issue, but without the clipping I seem to get a strange effect where the fixed images (from image banner) are visible behind certain sections like image with text, multicolumn, etc. This only happens with the zoom in effect selected on an image banner https://screenshot.click/20-05-6yruf-k9vdi.mp4 Are you seeing this?

Reply:
I had removed it here but it's necessary for the image banner section. On the other PR I had added it into the image banner CSS file and I just did this here too now, so hopefully that should be it 👌

Reply:
Gotcha, yeah all looks good now 👍